### PR TITLE
Adds buildnumber grunt task

### PR DESCRIPTION
**Pull request by scotthasbrouck**

### CODE REVIEW

**Comment by dmitrig01** [comment]
hmm... is there any way to just use the version number as the build number? or at least set both the version number and the build number to the same thing (using that perl regex)? just makes things simpler :-)

**Comment by scotthasbrouck** [comment]
Ah, yeah, we can do that. I'm heads down in revamping the storage class. So I'll make that change later tonight.

**Comment by dmitrig01** [comment]
sweet :D

**Comment by dmitrig01** [comment]
ping ^

File: package.json-e [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (dmitrig01) [inline package.json-e]: let's remove this file and ad to gitignore :-)

Comment (dmitrig01) [inline package.json-e]: then this pr is good to go

File: circle.yml [hunk 2]
```diff
@@ -28,5 +28,5 @@ dependencies:
 test:
   override:
     - ./lint.sh
+    - grunt buildnumber
```
Comment (dmitrig01) [inline circle.yml]: this can be removed :-)

**Comment by scotthasbrouck** [comment]
Cool, all set with those changes. I also removed most of the mobile tests to see if we have better luck with sauce labs timing out and giving false failures. From what I've read, SauceLabs has better desktop tests, and BrowserStack has better mobile tests because they use actual devices rather than emulators. I don't know if it's worth using both though.

**Pull request merged**